hexrd/xrdutil/phutil.py

```diff
@@ -460,7 +463,7 @@ def _infer_eta_shift(panel):
 
     eta_shift_dict = {
         'TARDIS': -np.radians(180),
-        'PXRDIP': -np.radians(90),
+        'PXRDIP': -np.radians(180),
```

Comment:
This one too. Maybe we can get rid of it also.

Reply:
Same as above.